src/app/courses/add-course-page/add-course-page.component.ts

```diff
@@ -1,8 +1,40 @@
 import { Component } from "@angular/core";
+import { ActivatedRoute, Router } from "@angular/router";
+import { ICourse, CourseService } from "../../core/index";
 
 @Component({
     selector: "app-add-course-page",
     templateUrl: "./add-course-page.component.html",
     styleUrls: ["./add-course-page.component.scss"]
 })
-export class AddCoursePageComponent {}
+export class AddCoursePageComponent {
+
+    private courseService: CourseService;
+    private router: Router;
+
+    public course: ICourse = {
+        id: NaN,
```

Comment:
better to use "new" string

Reply:
![image](https://user-images.githubusercontent.com/13545473/70309458-a3896f00-181e-11ea-8876-601de8a665eb.png)

it is a number on backend.